arboretum/ibm_cloud_databases/README.md

```diff
@@ -0,0 +1,126 @@
+# IBM Cloud Databases library
+
+The fetchers and checks contained within this `ibm_cloud_databases` category folder are
+common tests that can be configured and executed for the purpose of generating
+compliance reports and notifications using the [auditree-framework][].  They
+validate the configuration and ensure smooth execution of an auditree instance.
+See [auditree-framework documentation][] for more details.
+
+These tests are normally executed by a CI/CD system like
+[Travis CI](https://travis-ci.com/) as part of another project that uses this
+library package as a dependency.
+
+## Usage as a library
+
+See [usage][] for specifics on including this library as a dependency and how
+to include the fetchers and checks from this library in your downstream project.
+
+## Fetchers
+
+### IBM Cloud Databases List
+
+* Class: [DatabasesFetcher][fetch-databases]
+* Purpose: Store list of account-specific IBM Cloud databases to the evidence locker.
+* Behavior: Access [IBM Cloud API][ibm-cloud-api] details of the databases as a JSON. TTL for evidence is set to 1 day.
+* Configuration elements:
+  * `org.icd.list.accounts`
+    * Required
+    * List of objects representing the IBL Cloud accounts
+    * Each object must have the following fields
+      * `account_name` - an arbitrary name identifying the IBM Cloud account, used to match to the token provided in the
+      credentials file in order for the fetcher to retrieve content from IBM Cloud for that account.
+      * `resource_group_id` - the resource group associated with the above account name, output of [running the ibmcloud resource groups command][ic-resource-groups] while logged in to an IBM Cloud account.
+        ```
+        $ ibmcloud resource groups
+        Retrieving all resource groups under account fedcba1234567890abcdef0123456789 as dummy.user@ibm.com...
+        OK
+        Name      ID                                 Default Group   State
+        default   bcdef0123456789fedcba1234567890a   true            ACTIVE
+        ```
+
+* Example (required) configuration:
+
+  ```json
+  {
+    "org": {
+      "icd": {
+        "list": {
+          "accounts": [
+            {
+              "account_name": "prod_1234567",
+              "resource_group_id": [
+                                        "bcdef0123456789fedcba1234567890a",
+                                        ...
+                                   ]
+            }
+          ]
+        }
+      }
+    }
+  }
+  ```
+
+  The legacy config format where each account has a single resource_group_id is also supported
+
+  ```json
+  {
+    "org": {
+      "icd": {
+        "list": {
+          "accounts": [
+            {
+              "account_name": "prod_1234567",
+              "resource_group_id": "bcdef0123456789fedcba1234567890a"
+            }
+          ]
+        }
+      }
+    }
+  }
+  ```
```

Comment:
OK now it seems like you're introducing new functionality here?  If you're looking to support this new functionality and deprecate the old one where you have a single resource group as a string that's fine but you don't need to include the "backward compatibility" version in the docs.